examples/todo/src/main.rs

```diff
@@ -31,12 +31,18 @@ pub struct DbConn(SqliteConnection);
 struct Context<'a, 'b>{ msg: Option<(&'a str, &'b str)>, tasks: Vec<Task> }
 
 impl<'a, 'b> Context<'a, 'b> {
-    pub fn err(conn: &DbConn, msg: &'a str) -> Context<'static, 'a> {
-        Context{msg: Some(("error", msg)), tasks: Task::all(conn)}
+    pub fn err(conn: &DbConn, msg: &'b str) -> Context<'static, 'b> {
+        Self::raw(conn, Some(("error", msg)))
```

Comment:
I do think we should preserve the original error message; `tasks: Task::all(conn).unwrap_or_default()` is probably the clearest way.